docs/index.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: Serenity
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+[![CI Status](https://github.com/AY2021S1-CS2103T-W12-4/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103T-W12-4/tp/actions)
+[![codecov](https://codecov.io/gh/AY2021S1-CS2103T-W12-4/tp/branch/master/graph/badge.svg)](https://codecov.io/gh/AY2021S1-CS2103T-W12-4/tp)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**Serenity is a desktop application for managing your contact details.** While it has a *GUI*, most of the user interactions happen using a *CLI* (Command Line Interface).
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
+* If you are interested in using Serenity, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#2-quick-start).
 * If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```

Comment:
Missed out changing "AddressBook" to "Serenity" for the developer guide sentence